database/seeders/CountryTableSeeder.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class CountryTableSeeder extends Seeder
+{
+
+    /**
+     * Auto generated seed file
+     *
+     * @return void
+     */
+    public function run()
+    {
+        \DB::table('country')->delete();
+
+        \DB::table('country')->insert(array(
+            0 =>
+            array(
+                'id' => 1,
+                'slug' => 'usa',
+                'title' => 'United States of America',
+                'abbreviation' => 'US',
+                'capital' => 'Washington, D.C.',
+                'created_at' => '2021-03-03 02:08:35',
+                'updated_at' => '2021-03-03 02:08:35',
+            )
+        ));
```

Comment:
Let's move this to a migration instead of doing it in a seeder. Thanks!

Reply:
I'm going ahead with merging this code and we can move that to a migration in a later PR.

CC: @joshtorres @allibenson @arwaawan